src/swap/SwapScreen.tsx

```diff
@@ -132,6 +134,7 @@ export function SwapScreen() {
 
     if (parsedSwapAmount[Field.FROM].gt(maxFromAmount)) {
       setFromSwapAmountError(true)
+      showMaxCeloSwapWarning()
```

Comment:
keen for thoughts on this behaviour - i thought if the goal is to warn users about future implications of not having gas fees, then if they manually enter a swap amount that is too close to their absolute token balance then this warning would be helpful too?